pkgs/tools/misc/ltunify/default.nix

```diff
@@ -1,17 +1,17 @@
 { lib, stdenv, fetchFromGitHub }:
 
```

Comment:
imho that should be separate PR

Reply:
In principle, yes. These are however 2 tiny but related packages and in case we need to revert anything, it's in a separate commit.

Reply:
if somebody who can merge is happy with that, let's go